test/e2e/specs/editor/various/draggable-blocks.spec.js

```diff
@@ -18,7 +18,7 @@ test.describe( 'Draggable block', () => {
 		await admin.createNewPost();
 	} );
 
-	test( 'can drag and drop to the top of a block list', async ( {
+	test( 'can drag and drop to the top of a vertical block list', async ( {
```

Comment:
Note that changing the test's title will cause the reporter to open a new issue since it's the only id. I think it's okay though.